src/main/java/org/carlmontrobotics/robotcode2023/Constants.java

```diff
@@ -59,49 +66,30 @@ public static final class Arm {
 
         public static final double V_PER_NM = 0;
 
-        public static final double ARM_ANGLE_MIN = -3 * Math.PI / 2;
-        public static final double ARM_ANGLE_MAX = Math.PI / 2;
-        // how far it is from the min/max until it should stop the arm so that arm doesn't break robot
-        public static final double ARM_ANGLE_TOLERANCE = Math.PI / 2;
-        public static final double WRIST_ANGLE_MIN = -Math.PI;
-        public static final double WRIST_ANGLE_MAX = Math.PI;
-        // how far it is from the min/max until it should stop the wrist so that wrist doesn't break robot
-        public static final double WRIST_ANGLE_TOLERANCE = Math.PI / 6;
-
-        public static double MAX_FF_VEL = 0.1; // rad / s
-        public static double MAX_FF_ACCEL = 0.11; // rad / s
+        // TODO: Replace these values with Design's actual values
+        public static final double ARM_LOWER_LIMIT_RAD = -3 * Math.PI / 2;
+        public static final double ARM_UPPER_LIMIT_RAD = Math.PI / 2;
+        public static final double ARM_DISCONTINUITY_RAD = (ARM_LOWER_LIMIT_RAD + ARM_UPPER_LIMIT_RAD) / 2 - Math.PI;
+        public static final double WRIST_LOWER_LIMIT_RAD = -Math.PI;
+        public static final double WRIST_UPPER_LIMIT_RAD = Math.PI;
+        public static final double WRIST_DISCONTINUITY_RAD = (WRIST_LOWER_LIMIT_RAD + WRIST_UPPER_LIMIT_RAD) / 2 - Math.PI;
```

Comment:
Can you bound this from 0 to 2pi?

Reply:
I don't think that's needed. Is there a reason you want to do that?